features/step_definitions/cars_garage_steps.rb

```diff
@@ -24,9 +24,11 @@
 end
 
 When("I submit filled in new car form") do
-  fill_in "car_manufacturer", with: "Volkswagen"
-  fill_in "car_model", with: "Golfwagen LD"
-  fill_in "car_production_year", with: "2017"
+  fill_in 'car_custom_name', with: "Test car custom name"
+  select('2016', from: 'car_production_year')
+  select('Chevrolet', from: 'car_manufacturer')
```

Comment:
Could we please change it to some Mercedes-Benz?

Reply:
You're welcome.